database/postgres/dto/stream.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+from pydantic import BaseModel, Field
+from typing import Optional
+
+class StreamDTO(BaseModel):
+    stream_id: Optional[int] = Field(None)
+    title: Optional[str] = Field(None)
+    start_time: Optional[datetime] = Field(None)
+    sport_id: Optional[int] = Field(None)
+    stream_urls: Optional[list] = Field(None)
+
+    class Config:
+        from_attributes = True
+
+
+class StreamStatusDTO(BaseModel):
```

Comment:
```suggestion
class StreamStatusDTO(BaseModel):
    streams_status_id: Optional[int] = None
    status_id: Optional[int] = None
    stream_id: Optional[int] = None

    model_config = {"from_attributes": True}
```

can't you just do this? (It's supposed to be pydantic v2 approach while yours is v1. Not sure though)